pages/index.jsx

```diff
@@ -1,6 +1,8 @@
 import Head from 'next/head';
```

Comment:
There is an updated index file on `develop` branch. Please merge `develop` into your branch before create the PR.

Reply:
Done